submissions/description/server_security_misconfiguration/lack_of_security_headers/x_content_security_policy/template.md

```diff
@@ -0,0 +1,24 @@
+# Lack of X-Content-Security-Policy Header
+
+## Overview of the Vulnerability
+
+A lack of the HTTP response header for `X-Content-Security-Policy` can lead to sensitive user data being retrieved by an attacker and increases the attack surface for Cross-Site Scripting (XSS) and click-jacking attacks. There are multiple HTTP response headers used in communication between the server and client which can be implemented to improve security against well documented vulnerabilities. For example, the `X-Content-Security-Policy` security header allows admins to permissively control the types of resources allowed to load for a page.
+
+An advanced attacker can leverage a missing `X-Content-Security-Policy` header to launch XSS attacks and execute malicious code in a user’s browser.
+
+## Business Impact
+
+Not having a `X-Content-Security-Policy` header can lead to reputational damage and indirect financial loss to the business due to an advanced attacker’s ability to access data through a XSS attack. The degree of impact is dependent on the sensitivity of data being transmitted over the wire and the sophistication of the attacker’s abilities.
+
+## Steps to Reproduce
+
+1. Enable a HTTP interception proxy, such as Burp Suite or OWASP ZAP
+1. Navigate to the following endpoint using a browser: {{URL}}
+1. Capture the request using the HTTP interception proxy
```

Comment:
Clarifying a technical note, the request does not show HTTP headers, it's the response. Updating this to suit will help both readers of the report and the customer. 

1. Capture the request using the HTTP interception proxy and review the response

Reply:
Cheers @evildaemond, should be GTG